icetea_lib/TestBench/Resources.py

```diff
@@ -356,6 +355,13 @@ def _open_dut_connections(self, allocations):
                                      " started before we started reading? Try adding --reset"
                                      " to your run command.")
                 raise DutConnectionError("Dut cli failed to initialize within set timeout!")
+            if self._args.sync_start:
+                self._logger.info("Synchronizing the command line interface.")
+                try:
+                    self._commands.sync_cli(dut.index)
```

Comment:
actual function is missing, right ?

Reply:
Nope, the function is there. See the PR description.